Features/Standard/platformui.feature

```diff
@@ -0,0 +1,132 @@
+Feature: Login on PlatformUI
+
+    @javascript
+    Scenario: As a admin User, I want to login to PlatformUI
+        Given I go to PlatformUI app
+        And I fill in "Username" with "admin"
+        And I fill in "Password" with "publish"
+        When I click on the "Login" button
+        Then I should see "Welcome to eZ Platform" title
+
+    @javascript
+    Scenario: As a admin User, I want to login to PlatformUI
+        Given I go to PlatformUI app with username "admin" and password "publish"
+        Then I should see "Welcome to eZ Platform" title
+
+    @javascript
+    Scenario: As a admin User, I want to see Activity tab
+        Given I go to PlatformUI app with username "admin" and password "publish"
+        And I click on "Root location" link
+        When I click on the tab "Activity"
+        Then I should see "Activity content" text
```

Comment:
useless scenario, the activity tab is a manual testing thing, not a feature of the application